worlds/hk/__init__.py

```diff
@@ -626,6 +626,10 @@ def __init__(self, name, advancement, code, type, player: int = None):
             classification = ItemClassification.progression_skip_balancing
         elif type == "Charm" and name not in progression_charms:
             classification = ItemClassification.progression_skip_balancing
+        elif type in ("Map", "Journal"):
```

Comment:
The Map/Journal clause isn't necessary, since the 'else' clause handles all items that should be classed as filler.

Reply:
Quill and Hunter's Journal are specified in item_effects and are being flagged as advancement items as a result